### PR TITLE
Features/page designs

### PR DESCRIPTION
**Pull request by jackadrianglass**
I checked the output. Everything is good to go as far as I can tell

### CODE REVIEW

File: batan-client/components/ChatDisplay.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
                 class="list-item"
             >
                 <div class="user-img">
-                    <b-avatar src="$auth.user.image"/>
+                    <!-- <b-avatar :src="$auth.user.picture"/> -->
```
Comment (lhossackucalgary) [inline batan-client/components/ChatDisplay.vue:11]: Was this intended to be commented?

File: batan-client/pages/index.vue [hunk 2]
```diff
@@ -1,44 +1,71 @@
...
-  </div>
+  <b-container fluid class="container">
+    <b-row align-v="center">
+      <b-col>
+        <span class="title">Welcome to Batan.io! Create an account to play or sign in!</span>
+      </b-col>
+    </b-row>
+    <b-row align-v="center" class="logo-container">
+      <b-col>
+        <b-img src="@/static/batanLogo.png"/>
```
Comment (lhossackucalgary) [inline batan-client/pages/index.vue:10]: nit: Batan splash image is not reactive

**Review by lhossackucalgary — APPROVED** [review]
Looks good to me -- no breaking issues or warnings other than some that were there before; good start so we can work on individual components.

There are some odd artifacts (potentially component padding) in the game screen on Chrome & Firefox causing scroll bars for me. Might have better luck using grid layouts. 

Instead of 90% height for page containers, try height: calc(100vh - 3rem); (nav bar height is 3rem). Setting background colors on components helps with debugging layouts

File: batan-client/components/ChatDisplay.vue (hunk 1 above)
Comment (jackadrianglass) [reply batan-client/components/ChatDisplay.vue:11]: Yessir. It breaks the application when you login. I left it waiting on your stuff

File: batan-client/pages/index.vue (hunk 2 above)
Comment (jackadrianglass) [reply batan-client/pages/index.vue:10]: Everything is subject

**Pull request merged**